packages/zowe-explorer-api/src/vscode/ui/utils/HTMLTemplate.ts

```diff
@@ -14,28 +14,24 @@
 /**
  * HTML template that is compiled with Mustache to load a WebView instance at runtime.
  */
-const HTMLTemplate: string = `
+const HTMLTemplate: string = /*html*/ `
 <!DOCTYPE html>
 <html lang="en">
     <head>
         <meta charset="UTF-8">
         <title>{{ title }}</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta 
-            http-equiv="Content-Security-Policy" 
-            content="default-src 'none'; font-src data: {{ cspSource }}; img-src data: vscode-resource: https:; script-src {{#unsafeEval}}'unsafe-eval'{{/unsafeEval}} 'nonce-{{ nonce }}';
-            style-src {{ cspSource }} vscode-resource: 'unsafe-inline' http: https: data:;"
+        <meta
+            http-equiv="Content-Security-Policy"
+            content="default-src 'none'; font-src data: https:; img-src data: vscode-resource: https:; script-src 'nonce-{{ nonce }}';
```

Comment:
I believe we want to keep the cspSource in order for this to work over `http` (and other protocols?)

Reply:
Thanks for reverting back the changes.
If we get people asking about icons not loading, we can take a guess that it's because they are running on a different protocol (either http, or file)

This can also be addressed as part of:
- #3079